sanic/config.py

```diff
@@ -20,6 +26,63 @@ class Config:
 ▌     ▐                ▀▀▄▄▄▀
  ▀▀▄▄▀
 """
-    REQUEST_MAX_SIZE = 100000000  # 100 megababies
-    REQUEST_TIMEOUT = 60  # 60 seconds
-    ROUTER_CACHE_SIZE = 1024
+        self.REQUEST_MAX_SIZE = 100000000  # 100 megababies
+        self.REQUEST_TIMEOUT = 60  # 60 seconds
+        self.ROUTER_CACHE_SIZE = 1024
+
+    def __getattr__(self, attr):
+        return self[attr]
+
+    def __setattr__(self, attr, value):
+        self[attr] = value
+
+    def from_envvar(self, variable_name):
+        """Loads a configuration from an environment variable pointing to
+        a configuration file.
+        :param variable_name: name of the environment variable
+        :return: bool. ``True`` if able to load config, ``False`` otherwise.
+        """
+        config_file = os.environ.get(variable_name)
+        if not config_file:
+            raise RuntimeError('The environment variable %r is not set and '
+                               'thus configuration could not be loaded.' %
+                               variable_name)
+        return self.from_pyfile(config_file)
+
+    def from_pyfile(self, filename):
+        """Updates the values in the config from a Python file. Only the uppercase
+        varibales in that module are stored in the config.
+        :param filename: an absolute path to the config file
+        """
+        module = types.ModuleType('config')
+        module.__file__ = filename
+        try:
+            with open(filename) as config_file:
+                exec(compile(config_file.read(), filename, 'exec'),
```

Comment:
I tried using importlib, however it fails when the config file does not have the .py suffix. Anyway, whether we are loading with importlib or with compile/exec does not really matter, in both cases we're executing python code. Or is there any benefit I'm missing?